source/options.html

```diff
@@ -8,25 +8,21 @@ <h2 class="only-firefox">Options</h2>
 
 	<p>
 		<label>
-			<strong>Custom CSS</strong> (useful to undo unwanted styles)<br>
-			<textarea name="customCSS" rows="2" spellcheck="false"></textarea>
+			<strong>Features to disable</strong>, by <a href="https://github.com/sindresorhus/refined-github/tree/master/source/features" target="_blank">filename</a><br>
+			<textarea name="disabledFeatures" rows="2" placeholder="For example: &#10;mark-unread hide-own-stars" spellcheck="false"></textarea>
 		</label>
 	</p>
 
-	<hr>
-
 	<p>
 		<label>
-			<strong>Features to disable</strong>, by <a href="https://github.com/sindresorhus/refined-github/tree/master/source/features" target="_blank">filename</a> (not advised)<br>
-			<textarea name="disabledFeatures" rows="2" placeholder="For example: &#10;mark-unread hide-own-stars" spellcheck="false"></textarea>
+			<strong>Custom CSS</strong> (useful to undo unwanted styles)<br>
+			<textarea name="customCSS" rows="2" spellcheck="false"></textarea>
 		</label>
 	</p>
 
-	<hr>
-
 	<p>
 		<label>
-			<strong>Personal token</strong>, <a href="https://github.com/settings/tokens/new?description=Refined%20GitHub&scopes=repo" target="_blank">found here</a> (for high rate API calls and private repos)<br>
+			<strong>Personal token</strong>, <a href="https://github.com/settings/tokens/new?description=Refined%20GitHub&scopes=repo" target="_blank">found here</a> (some features use the API)<br>
```

Comment:
Can we keep the call out for private repos?

Reply:
I think `API` covers it all. Only features that use the API (whether private or not) require the token. It'd be like saying "this is for drinks and water"